shared/src/main/scala/squants/mass/Mass.scala

```diff
@@ -171,6 +182,45 @@ object SolarMasses extends MassUnit {
   val symbol = "M☉"
 }
 
+object ElectronVolt extends MassUnit {
```

Comment:
This is problematic in general. Having the same name will create problems either on the REPL or other cases. I'm kind of leaning to use `ElectronVolt` for `Energy` and use something like `ElectronVoltMass` in this case and similar

What do you think?